src/scales/scale.time.js

```diff
@@ -66,19 +66,18 @@ function sorter(a, b) {
 }
 
 function arrayUnique(items) {
-	const hash = {};
-	const out = [];
-	let i, ilen, item;
+	const set = new Set();
+	let i, ilen;
 
 	for (i = 0, ilen = items.length; i < ilen; ++i) {
-		item = items[i];
-		if (!hash[item]) {
-			hash[item] = true;
-			out.push(item);
-		}
+		set.add(items[i]);
+	}
+
+	if (set.size === ilen) {
+		return items;
 	}
 
-	return out;
+	return Array.from(set);
```

Comment:
This isn't available in IE11

Reply:
Missed that, thanks! 
I'll update (later)